static/src/js/actions/focusedCollection.js

```diff
@@ -234,6 +234,17 @@ export const getFocusedCollection = () => async (dispatch, getState) => {
           versionId
         } = collection
 
+        // Look and see if there are any gibs tags
+        // If there are, check to see if the colormaps associated with the productids in the tags exists.
+        // If they don't we call an action to pull the colorMaps and add them to the metadata.colormaps
+        if (tags && 'edsc.extra.serverless.gibs' in tags) {
```

Comment:
I think we have a helper util function meant to grab data from a tag. We should probably use that here.